### PR TITLE
Ftr: protocol_conf default values

### PR DESCRIPTION
**Pull request by joeyzhouy**
Ftr: issue #215

### CODE REVIEW

**Comment by AlexStocks** [comment]
@fangyincheng pls check this pr. if there is no problems, u can merge it.

File: protocol/dubbo/config.go [hunk 1]
```diff
@@ -90,6 +90,57 @@ type (
...
+			TcpNoDelay:       true,
+			TcpKeepAlive:     true,
+			KeepAlivePeriod:  "120s",
+			TcpRBufSize:      262144,
+			TcpWBufSize:      65536,
+			PkgWQSize:        512,
+			TcpReadTimeout:   "1s",
+			TcpWriteTimeout:  "5s",
+			WaitTimeout:      "1s",
+			MaxMsgLen:        2014,
```
Comment (pantianying) [inline protocol/dubbo/config.go]: why use 2014?  and I think 2014 is too short.
I use 102400

File: protocol/dubbo/config.go [hunk 2]
```diff
@@ -90,6 +90,57 @@ type (
...
+			TcpNoDelay:       true,
+			TcpKeepAlive:     true,
+			KeepAlivePeriod:  "120s",
+			TcpRBufSize:      262144,
+			TcpWBufSize:      65536,
+			PkgWQSize:        512,
+			TcpReadTimeout:   "1s",
+			TcpWriteTimeout:  "5s",
+			WaitTimeout:      "1s",
+			MaxMsgLen:        1024,
```
Comment (pantianying) [inline protocol/dubbo/config.go]: MaxMsgLen is too small

File: protocol/dubbo/config.go [hunk 3]
```diff
@@ -90,6 +90,57 @@ type (
...
+	return ClientConfig{
+		ReconnectInterval: 0,
+		ConnectionNum:     16,
+		HeartbeatPeriod:   "5s",
+		SessionTimeout:    "20s",
+		PoolSize:          64,
+		PoolTTL:           600,
+		GrPoolSize:        1200,
+		QueueLen:          64,
+		QueueNumber:       60,
```
Comment (pantianying) [inline protocol/dubbo/config.go]: this size of config may cause higher CPU，this is my config I use better
image

File: protocol/dubbo/config.go (hunk 2 above)
Comment (joeyzhouy) [reply protocol/dubbo/config.go]: Configuration from example.
If the configuration is not reasonable, I can change it.

File: protocol/dubbo/config.go (hunk 1 above)
Comment (joeyzhouy) [reply protocol/dubbo/config.go]: fix

File: protocol/dubbo/config.go (hunk 2 above)
Comment (joeyzhouy) [reply protocol/dubbo/config.go]: fix

File: protocol/dubbo/config.go (hunk 3 above)
Comment (joeyzhouy) [reply protocol/dubbo/config.go]: Thank you for sharing your ideas

**Pull request merged**